text/0000-struct-closure.md

```diff
@@ -0,0 +1,223 @@
+- Feature Name: `struct_closure`
+- Start Date: 2022-08-05
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+[summary]: #summary
+
+Concrete Closure Types
+
+# Motivation
+[motivation]: #motivation
+
+This proposal exists entirely out of spite for the `char::is_ascii*` family of
+functions.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+Closures in Rust are represented by the struct `core::ops::Closure`, while
```

Comment:
I [do not see such a struct](https://doc.rust-lang.org/nightly/core/?search=core%3A%3Aops%3A%3AClosure#method.clone). Can you please link to it?

Reply:
The RFC template says to write here "as if [the proposal] was already included in the language". This is that.